services/integrate-android-app-with-custom-service.adoc

```diff
@@ -0,0 +1,70 @@
+= Integrate an Android Application with Custom Services Running in OpenShift
+
+This document describes the steps needed to configure the Aerogear Android SDK to allow your Android app to interact with:
+
+* Your own backend services in OpenShift
+* External backend services
+
+== Prerequisites
+
+* link:./create-android-client-on-openshift.adoc[Create an Android Mobile Client in OpenShift]
+
+== Procedure
+
+For demonstration, this guide uses the example Python application which can be found in the OpenShift Service Catalog. The same steps can be followed any HTTP web service running in OpenShift that has a publicly accessible route.
+
+image:./images/custom-service-py.png[Custom Service and Android Client in OpenShift]
```

Comment:
This image looks out of place here. Should it be at the end of the next section?

Reply:
I guess the idea with that image was to show that we've already got a mobile client and a web service of some sort. Perhaps it's not the best though